CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ##### Enhancements
 
+* Replace git-based MasterSource with CDN-based TrunkSource
+  [igor-makarov](https://github.com/igor-makarov)
```

Comment:
I know its somewhat counter-intuitive but there is no issue in Core for this, either create an issue and add a link here to it or then just add a link to PR #552.

Reply:
Shouldn't this just link to CocoaPods#8923?